### ISSUE TITLE
WCAG 2.2 contrast ratio - non compliant for text?

### ISSUE THREAD

**Issue by robnemeth** [alphagov/guide-to-wcag#185]
The examples given on multiple pages do not meet WCAG 2.2.    For example:

https://design-system.service.gov.uk/components/date-input/default/

The contract between the text  "When was your passport issued" and "For example, 27 3 2007" is 2.77.1 

This is based on #0b0c0c to #505a5f which are listed as text colours on https://design-system.service.gov.uk/styles/colour/

**Comment by andrewhick** [comment]
Hi @robnemeth , thank you for letting us know about contrast ratios for hint text.

Your comment seems to relate to the Design System which is separate from this repository - this repository only covers the [Guide to WCAG](https://alphagov.github.io/guide-to-wcag/). To raise an issue on the Design System, I would recommend that you contact the email address under "Feedback and contact information" on the [Design System accessibility statement](https://design-system.service.gov.uk/accessibility-statement/).

From your description I'm not convinced that the Date Input example fails WCAG, because there is enough contrast between both sets of text and their background. For example, the grey text has a contrast ratio of around 7:1 with the white background. There is no requirement for text to have a particular contrast ratio with other text, unless information is being conveyed only using a change of colour. I don't think that is the case here.

I'll keep the issue open for a few days in case you disagree but hopefully the Design System team can help if needed. Many thanks, Andrew
